docs/preset-flow.md

```diff
@@ -99,7 +99,18 @@ class A {
 
 Added in: `v7.21.4`
 
-When it is set to `true`, Babel will apply the flow transform to all extensions. When it is set to `false`, Babel
-will avoid the flow transform for `*.tsx` files.
+When it is set to `true`, Babel will apply the flow transform to all extensions. When it is set to `false`, Babel will avoid the flow transform for `*.tsx` files.
 
 > You can read more about configuring preset options [here](https://babeljs.io/docs/en/presets#preset-options)
```

Comment:
Nit: Can you insert the new option docs before this line? It is the epilogue of preset docs.